### PR TITLE
Add support locationtech

### LINKED ISSUE #2404: Please update to current JTS version (org.locationtech.jts:jts-core:1.16.0)
Please exchange the old jts dependency with the maintained successor:
https://mvnrepository.com/artifact/org.locationtech.jts/jts-core/1.16.0

### PR DESCRIPTION
**Pull request by NikitaKochkurov**
Fixes #2404

### CODE REVIEW

**Comment by jwgmeligmeyling** [comment]
Thanks! I'm going to review this this week.

**Comment by Firestorm87** [comment]
I am currently looking for exact this locationtech support.
Please tell me if i can help you to integrate this.

**Comment by NikitaKochkurov** [comment]
@Firestorm87  Yes! @jwgmeligmeyling  Why not!?

**Comment by jwgmeligmeyling** [comment]
If it wasn't clear already by assigning this to the milestone: this will be integrated in to the next version of QueryDSL 5.x.

We first have some bigger issues for 5.x to get out of the way. 5.x won't be released without this though.

My ETA for QueryDSL 5 is a month at most.

**Comment by NikitaKochkurov** [comment]
@jwgmeligmeyling  ok, no problem.

File: querydsl-spatial2/pom.xml [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.querydsl</groupId>
+        <artifactId>querydsl-root</artifactId>
+        <version>4.4.1-SNAPSHOT</version>
```
Comment (jwgmeligmeyling) [inline querydsl-spatial2/pom.xml:10]: Now the PR is up to date, this should be updated  to `5.0.0-SNAPSHOT`. Then the build will work again.

**Comment by jwgmeligmeyling** [comment]
Thanks for updating the PR. Sorry for the delay!

**Pull request closed without merging**